lib/internal/bootstrap_node.js

```diff
@@ -249,6 +250,21 @@
     });
   }
 
+  function setupInspectorAPI() {
+    const addCommandLineAPIMethod = process.inspector.addCommandLineAPIMethod;
+    addCommandLineAPIMethod('require', function require(request) {
```

Comment:
is this callback called synchronously? if not, I need to inspect the startup timing a bit more